Example/PaymentSheet Example/PaymentSheet Example/CustomerSheetTestPlayground.swift

```diff
@@ -78,6 +79,19 @@ struct CustomerSheetTestPlayground: View {
         }
     }
 
+    var merchantCountryBinding: Binding<CustomerSheetTestPlaygroundSettings.MerchantCountry> {
+        Binding<CustomerSheetTestPlaygroundSettings.MerchantCountry> {
+            return playgroundController.settings.merchantCountryCode
+        } set: { newCountry in
+            // Reset customer id if country changes
+            if playgroundController.settings.merchantCountryCode.rawValue != newCountry.rawValue {
+                playgroundController.settings.customerId = nil
```

Comment:
There shouldn't be any need to set this to nil, right?

Reply:
Just tested this, yep just setting to new customer is sufficient.